Engine/Glad.cmake

```diff
@@ -0,0 +1,14 @@
+# Glad
```

Comment:
I think that here is okay to have a .cmake as glad is a dependency, but you can also convert it to CMakeLists.txt as it is a default way

Reply:
And it also could mess with PRIVATE/PUBLIC